server/src/main/java/io/crate/execution/engine/fetch/FetchCollector.java

```diff
@@ -103,7 +103,7 @@ public StreamBucket collect(IntArrayList docIds) {
                     }
                     setNextDocId(readerContext, docId - subReaderContext.docBase);
                 } catch (IOException e) {
-                    Exceptions.rethrowRuntimeException(e);
+                    throw new UncheckedIOException(e);
```

Comment:
Is this correct? just double checking.

Reply:
I don't think it breaks anything and `UncheckedIOException` is more informative than wrapping it in `RuntimeException`

Reply:
thx